tests/test_querysets.py

```diff
@@ -220,22 +220,22 @@ def test_aggregate_no_grouping(self):
     def test_aggregate_with_filter(self):
         # When filter comes before aggregate
         qs = Person.objects_in(self.database).filter(first_name='Warren').aggregate(average_height='avg(height)', count='count()')
-        print qs.as_sql()
+        print(qs.as_sql())
         self.assertEquals(qs.count(), 1)
         for row in qs:
             self.assertAlmostEqual(row.average_height, 1.675, places=4)
             self.assertEquals(row.count, 2)
         # When filter comes after aggregate
         qs = Person.objects_in(self.database).aggregate(average_height='avg(height)', count='count()').filter(first_name='Warren')
-        print qs.as_sql()
+        print(qs.as_sql())
```

Comment:
Can you please check this out one more time? I've included unicode_literals everywhere, even in tests, and fixed errors which were caused because of it.